### PR TITLE
Fix PPU Interpreter config

### PR DESCRIPTION
**Pull request by vlj**
This PR cherries pick 2 commits from Zangetsu master branch.
These commits make possible to use ppu interpreter2 and llvm recompiler.

### CODE REVIEW

File: rpcs3/Ini.h [hunk 1]
```diff
@@ -472,11 +443,8 @@ class Inis
 	{
 		switch (code)
 		{
-		case 0: return "WARP";
-		case 1: return "Default";
-		case 2: return "Renderer 0";
-		case 3: return "Renderer 1";
-		case 4: return "Renderer 2";
+		case 0: return "Default";
```
Comment (Nekotekina) [inline rpcs3/Ini.h]: What happened there?

**Comment by Nekotekina** [comment]
Apparently Oil says he'll fix things soon.

**Comment by vlj** [comment]
Yes but I need the changes now, and it's already available.

File: rpcs3/Ini.h (hunk 1 above)
Comment (vlj) [inline rpcs3/Ini.h]: The adaptater's name did change a while back, but I forgot to fix it in the log part.

**Comment by Zangetsu38** [comment]
@Nekotekina Neko, I fixed the problem that night, immediately after the PR https://github.com/RPCS3/rpcs3/pull/1272, so again, where is the problem?
Whether it's me or 01L that do so, it is similar, and I I is set immediately.
My code does not break anything, I do not see why my "PR" worry like that.

**Pull request closed without merging**